methods.rb

```diff
@@ -0,0 +1,96 @@
+module Enumerable
+  def my_each
+    for i in self
+      yield(i) if block_given?
+    end
+  end
+
+  def my_each_with_index
```

Comment:
Additional Requirements for this method:
it should:
- return an Enumerator if no block is given